test/framework/integration_framework/fake_peer/fake_peer.hpp

```diff
@@ -66,7 +69,8 @@ namespace integration_framework {
         shared_model::interface::AbstractTransportFactory<
             shared_model::interface::Transaction,
             iroha::protocol::Transaction>;
-    using YacStatePtr =
+    using MstMessagePtr = std::shared_ptr<MstMessage>;
```

Comment:
this type I guess can be shared via defining at the MstMessage file

Reply:
I would postpone this fix as in the next PR I move it from here anyway